src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/DependenciesProjectTreeProvider.cs

```diff
@@ -560,7 +560,6 @@ public IRule GetRule(IDependency dependency, IProjectCatalogSnapshot catalogs)
             var schema = browseObjectsCatalog.GetSchema(dependency.SchemaName);
             var itemSpec = string.IsNullOrEmpty(dependency.OriginalItemSpec) ? dependency.Path : dependency.OriginalItemSpec;
             var context = ProjectPropertiesContext.GetContext(UnconfiguredProject,
```

Comment:
If we pass the file, CPS uses it's name as a suffix appended to project folder when it tries to resolve a project path: http://index/?rightProject=Microsoft.VisualStudio.ProjectSystem&file=ProjectProperties%5cContext%5cProjectPropertiesContext.cs&line=156 which does not exist and CPS by some reason attempts to create folders  - which is a bug on their side , i don't think they intended to create folders .